plugins/field-colour-hsv-sliders/package.json

```diff
@@ -59,6 +59,6 @@
     "extends": "@blockly/eslint-config"
   },
   "engines": {
-    "node": ">=8.0.0"
```

Comment:
Same question related to this change?

Reply:
Hi @debaraj-barua!
I updated the Node.js version to address issues on my local host. The suggestion was to upgrade to 14+, but I chose version 16 to resolve the problem, resulting in unintended package changes.

I apologize for any confusion. This is my first experience with an open-source project, and If you have any questions or suggestions, please feel free to let me know. Thx!

Reply:
No need to apologize :) I was just wondering about the change. This seems like a good idea to upgrade node, but it might have been set to `8` for backwards compatibility. I am unsure of this.

One of the maintainers might be able to actually answer my question, if this change is required/desirable.

cc: @BeksOmega

Reply:
Hello! We don't want to bump the minimum node version at the moment.

We have a separate [issue](https://github.com/google/blockly-samples/issues/1531) filed for investigating this and possibly upgrading it. So we're planning to fix that separately =)